polygon-id/idScanPassport/idScanPassport.json

```diff
@@ -19,13 +19,14 @@
         "id": {
           "description": "Stores the DID of the subject that owns the credential",
           "title": "Credential subject ID",
-          "format": "uri",
-          "type": "string"
+          "type": "string",
```

Comment:
if you generate a scheam here: https://schema-builder.polygonid.me/builder
then this is the ID that is generated, i'd like to keep it this way
```
 "id": {
        "description": "Stores the DID of the subject that owns the credential",
        "title": "Credential subject ID",
        "format": "uri",
        "type": "string"
      }
```

Reply:
id should now be changed throughout